lib/agent.js

```diff
@@ -345,9 +345,9 @@ if (process.platform === 'win32') {
   var RET_ERR_BINSTDIN = 13;
   var RET_ERR_BINSTDOUT = 14;
   var RET_ERR_BADLEN = 15;
-
+  var RAWPATH = path.resolve(__dirname, '..', 'util/pagent.exe');
   var ERROR = {};
-  var EXEPATH = path.resolve(__dirname, '..', 'util/pagent.exe');
+  var EXEPATH = RAWPATH.includes('app.asar') ? RAWPATH.replace('app.asar', 'app.asar.unpacked') : RAWPATH;
```

Comment:
This check looks incorrect. What if the path already is inside `app.asar.unpacked`? It will hit this path and result in `app.asar.unpacked.unpacked`

Reply:
I mostly added this code as a proof of concept to start the discussion. I agree it needs a little tuning, but figured you might want a more general solution.